packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx

```diff
@@ -0,0 +1,191 @@
+// @ts-strict-ignore
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏anything we could do to make this file strict? Ideally we shouldn't create _new_ code with this flag.